build.sh

```diff
@@ -887,15 +891,14 @@ case "$COMMAND" in
     # CocoaPods
     ######################################
     "cocoapods-setup")
+        if [ ! -d core ]; then
+          # core directory won't exist if running `pod spec lint` locally
```

Comment:
This comment doesn't really make any sense. This command is the thing that downloads core when installing the pod normally, so of course the core directory won't exist.